src/sentry/web/frontend/reactivate_account.py

```diff
@@ -13,7 +12,6 @@ class ReactivateAccountView(BaseView):
     auth_required = False
 
     @never_cache
-    @transaction.atomic
```

Comment:
Do we need to wrap the handler code below in a transaction since we're removing this? Or was this deco just completely unnecessary?

Reply:
Nope.  looking through the code it honestly never needed this wrapper.